boards/raspberrypi/common/rpi_pico-pinctrl-common.dtsi

```diff
@@ -23,14 +23,6 @@
 		};
 	};
 
-	i2c1_default: i2c1_default {
-		group1 {
-			pinmux = <I2C1_SDA_P6>, <I2C1_SCL_P7>;
-			input-enable;
-			input-schmitt-enable;
-		};
-	};
-
```

Comment:
The [pico-examples](https://github.com/raspberrypi/pico-examples) use the same pinmux for `I2C1` as was defined before this PR, i.e. using GPIO6 and GPIO7. I think that the vendor's examples should be considered a strong enough precedent.

Reply:
@ajf58 @kartben 

https://github.com/zephyrproject-rtos/zephyr/pull/45131#discussion_r872782328

This topic has already been discussed and agreed upon, and it violates that, so it needs to be restored.
If you would like to raise a separate issue to re-enable it, that is welcome.